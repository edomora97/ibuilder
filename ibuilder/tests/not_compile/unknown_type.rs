struct Foo {
    field: Bar,
}

fn main() {
    Foo::builder();
}
