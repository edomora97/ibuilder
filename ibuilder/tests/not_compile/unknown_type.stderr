error[E0412]: cannot find type `Bar` in this scope
 --> $DIR/unknown_type.rs:2:12
  |
2 |     field: Bar,
  |            ^^^ not found in this scope

error[E0599]: no function or associated item named `builder` found for struct `Foo` in the current scope
 --> $DIR/unknown_type.rs:6:10
  |
1 | struct Foo {
  | ---------- function or associated item `builder` not found for this
...
6 |     Foo::builder();
  |          ^^^^^^^ function or associated item not found in `Foo`
